troubleshooting.md

```diff
@@ -0,0 +1,550 @@
+# Troubleshooting Guide for BM IPI Deployments
+
+
+* [Troubleshooting Overview](#troubleshooting-overview)
+
+* [Bootstrap VM issues](#bootstrap-vm-issues)
+
+* [Inspecting Logs](#inspecting-logs)
+
+* [Following the Installation](#following-the-installation) 
+
+* [Post install pod errors](#post-install-pod-errors)
+
+* [NTP out of sync](#ntp-out-of-sync)
+
+* [Registry Issues](#registry-issues) 
+
+* [Cleaning up previous installations](cleaning-up-previous-installations)
+
+* [Misc Issues](#misc-issues) 
+
+
+# Troubleshooting Overview
+
+When attempting to deploy Installer Provisioned Infrastructure (IPI) of OpenShift on Bare Metal (BM) you may run into a situation where you need to troubleshoot your environment. This document provides troubleshooting guidance and tips in solving common issues that may arise. 
+
+# Troubleshooting Flow 
+
+Prior to troubleshooting your environment, it is critical to understand the overall flow of the IPI on BM installation. The diagram below is a troubleshooting flowchart providing a step-by-step breakdown of where to start debugging your environment.
+
+![Flow-Diagram](https://github.com/DirectedSoul1/baremetal-deploy/blob/master/images/OpenShift%20Baremetal%20Deployment%20Troubleshooting%20Flow.png)
```

Comment:
File should be in this repository, not external file

Reply:
This file is in my own repo not an external one, which repo do you suggest please?

Reply:
Additional file in this same repo :-)